### DISCUSSION TITLE
VITS recipe wont fetch wav files despite path seems correct

### DISCUSSION THREAD

**Discussion by AnnGL93** [coqui-ai/TTS#1654] [General Q&A]
Hello, I'm new to TTS but have been following threads and tutorials for hours and although I have used the train_vits_tts.py file for configuring, just changing 

dataset_config = BaseDatasetConfig(name="mailabs", meta_file_train="metadata.csv", path=os.path.join(output_path, mailabs_path), language="es")

I get the following outcome (0 files found). As you can see in the picture, the path seems correct. Could someone please give me some light in what I'm doing wrong? Thank you in advance
![TTs VIT](https://user-images.githubusercontent.com/107587244/173938183-4a8e52a7-529f-486b-9c3b-198e572d86fa.jpg)

**Comment by p0p4k** [comment]
Did you fix this?

**Reply by AnnGL93** [reply]
Not yet. I think it has something to do with other libraries or the type of file. Though I've been using .wav files and ljspeech with a .csv file for metadata (As I found in an example), different combinations in the config file will continue to through other errors like "index out of range" in the formatters.py file. I guess this reflects the same problem (cannot read the wav files). If you find any solution please let me know.
